### DISCUSSION TITLE
Metric for object count in a bucket

### DISCUSSION THREAD

**Discussion by amir-latifi** [seaweedfs/seaweedfs#5949] [General]
Hi, 
Is there any prometheus metric for count of objects inside a bucket? Something like "minio_bucket_usage_object_total" in minio.
And is there any metric for size distribution of objects like "minio_bucket_objects_size_distribution" in minio?
If no, is there any plan to add these metrics? 
Thank you,

**Comment by chrislusf** [comment]
each bucket is one collection. there are collection level metrics for total size, but not for count.

no immediate plan. PRs are welcome.

https://github.com/search?q=repo%3Aseaweedfs%2Fseaweedfs%20VolumeServerDiskSizeGauge&type=code
